aries_cloudagent/messaging/schemas/routes.py

```diff
@@ -16,66 +16,38 @@
 class SchemaSendRequestSchema(Schema):
     """Request schema for schema send request."""
 
-    schema_name = fields.Str(
-        required=True,
-        description="Schema name",
-        example="prefs",
-    )
+    schema_name = fields.Str(required=True, description="Schema name", example="prefs",)
     schema_version = fields.Str(
-        required=True,
-        description="Schema version",
-        **INDY_VERSION
+        required=True, description="Schema version", **INDY_VERSION
     )
     attributes = fields.List(
-        fields.Str(
-            description="attribute name",
-            example="score",
-        ),
+        fields.Str(description="attribute name", example="score",),
         required=True,
-        description="List of schema attributes"
+        description="List of schema attributes",
```

Comment:
comma consistency

Reply:
That's an annoying issue with the Black autoformatter, it will insert the trailing comma when a call spans multiple lines, but doesn't remove it when it later shortens it to one line.